src/App.tsx

```diff
@@ -117,7 +115,7 @@ function App() {
               href="https://github.com/CovidEngine/vaxxnzlocations"
               target="_blank"
               rel="noreferrer"
-              onClick={() => enqueueAnalyticsEvent('Raw data clicked')}
+              onClick={() => enqueueAnalyticsEvent("Raw data clicked")}
```

Comment:
@MinhPhu0304 I think you might be missing Prettier in your editor, it would automatically fix these styling issues